ssr/src/page/post_view/bet.rs

```diff
@@ -176,54 +175,81 @@ fn HNButtonOverlay(
+                        <button disabled=running on:click=move |_| coin.update(|c| *c = c.wrapping_next())>
+                            <Icon class="justify-self-end text-2xl text-white" icon=icondata::AiUpOutlined />
+                        </button>
+                    </div>
+                    <div class="flex flex-row gap-6 justify-center items-center w-full touch-manipulation">
+                        <HNButton disabled=running bet_direction kind=BetKind::Hot />
+                        <button disabled=running on:click=move |_| coin.update(|c| *c = c.wrapping_next())>
+                            <CoinStateView
+                                disabled=running
+                                class="w-12 h-12 md:w-14 md:h-14 lg:w-16 lg:h-16 drop-shadow-lg"
+                                coin
+                            />
+                        </button>
+                        <HNButton disabled=running bet_direction kind=BetKind::Not />
+                    </div>
+                    // Bottom row: Hot <down arrow> Not
+                    // most of the CSS is for alignment with above icons
+                    <div class="flex gap-6 justify-center items-center pt-2 w-full text-base font-medium text-center md:text-lg lg:text-xl touch-manipulation">
+                        <p class="w-14 md:w-16 lg:w-18">Hot</p>
+                        <div class="flex justify-center w-12 md:w-14 lg:w-16">
+                            <button disabled=running on:click=move |_| coin.update(|c| *c = c.wrapping_prev())>
+                                <Icon class="text-2xl text-white" icon=icondata::AiDownOutlined />
+                            </button>
+                        </div>
+                        <p class="w-14 md:w-16 lg:w-18">Not</p>
+                    </div>
+                    <ShadowBg />
+                       <Show when=move || running.get() >
+        <div class="flex fixed bottom-8 left-1/2 justify-center items-center mb-8 w-full transform -translate-x-1/2">
+            <div class="flex flex-col gap-1 justify-center items-center p-4 w-1/4 h-1/4 text-white rounded-md bg-black/50">
+                <div class="flex flex-col gap-1 justify-center items-center p-1 w-full shadow-sm">
+                    <p class="text-center text-white rounded-full bg-black/15 ps-2">
+                        You staked {tracked_coin()} tokens on
+                        {if let Some(bet_kind) = bet_direction() {
+                            if bet_kind == BetKind::Hot {
+                                "Hot"
+                            } else {
+                                "Not"
+                            }
+                        } else {
+                            "?"
+                        }}. Result is still pending
+                    </p>
+                </div>
             </div>
-            <p class="w-14 md:w-16 lg:w-18">Not</p>
         </div>
-        <ShadowBg />
-    }
+    </Show>
```

Comment:
Can you format the DOM properly?

Reply:
yes , onto some thing right now.
next thing is this